### PR TITLE
added files generated by provider generator

### PR DESCRIPTION
**Pull request by durandom**
Files in here are 100% generated by the provider generator.
They should make the extraction PR https://github.com/ManageIQ/manageiq/pull/12274 green.

If you find anything odd or not good looking I try to fix it in the generator

@Fryguy @djberg96 @bronaghs @juliancheal @blomquisg

### CODE REVIEW

**Comment by chessbyte** [comment]
@Fryguy @durandom was Travis supposed to run here?

**Comment by durandom** [comment]
I guess travis is not setup yet.

**Comment by juliancheal** [comment]
@durandom You may as well add the commits from https://github.com/ManageIQ/manageiq-providers-amazon/commit/536ce7a2ba6dfc04cc832c560731bbb335825d36

As Codeclimate has changed.

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+begin
+  require 'bundler/setup'
+rescue LoadError
+  puts 'You must `gem install bundler` and `bundle install` to run rake tasks'
```
Comment (djberg96) [inline Rakefile]: Do you want to re-raise the error here? Or exit? Also, maybe substitute `puts` with `warn`.

Comment (djberg96) [inline Rakefile]: Then it needs work ;)

**Comment by djberg96** [comment]
I vote to merge. Let's get going.

File: lib/manageiq/providers/azure.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+require "manageiq/providers/azure/engine"
+
+module Manageiq
```
Comment (Fryguy) [inline lib/manageiq/providers/azure.rb]: Minor typo, but this should be `ManageIQ`

Comment (Fryguy) [inline lib/manageiq/providers/azure.rb]: Going to merge as is... @durandom Please fix this in a follow up PR and in the generator as well

Comment (durandom) [inline lib/manageiq/providers/azure.rb]: Oops, are modules case insensitve?
Will fix this...

**Pull request merged**